tests/src/analysis/testqgsprocessing.cpp

```diff
@@ -860,10 +860,16 @@ void TestQgsProcessing::compatibleLayers()
   QgsMeshLayer *m2 = new QgsMeshLayer( fm.filePath(), "mA", "mdal" );
   QVERIFY( m2->isValid() );
 
```

Comment:
Again, this should use ept files and be conditional on WITH_EPT

Reply:
Done